includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -336,6 +344,22 @@ public function process_subscription_payment( $amount, $renewal_order, $retry =
 				if ( is_callable( [ $renewal_order, 'save' ] ) ) {
 					$renewal_order->save();
 				}
+			} elseif ( $this->must_authorize_off_session( $response ) ) {
+				$charge_attempt_at = $response->next_action->card_await_notification->charge_attempt_at;
```

Comment:
According to the [documentation](https://stripe.com/docs/india-recurring-payments#trigger-predebit-notification) this should be changed:

```suggestion
				$charge_attempt_at = $response->processing->card->customer_notification->completes_at;
```

However, the API responses I'm getting from the API don't match the documentation so I've changed it to work even if it doesn't seem to be correct. I've reached out to Stripe in p1678862901758999-slack-C9976E5MJ to ask for clarification.

Reply:
Committed the suggestion since it's more likely people will have this version of the API than what I'm seeing personally.